services/archiver/tablearchiver/tablearchiver.go

```
@@ -95,7 +95,7 @@ func (jsonArchiver *TableJSONArchiver) Do() (location string, err error) {
 		jsonBytes = jsonBytes[1 : len(jsonBytes)-1]                                 // stripping starting '[' and ending ']'
 		jsonBytes = append(jsonBytes, '\n')                                         // appending '\n'
 
-		gzWriter.Write(jsonBytes)
+		_, _ = gzWriter.Write(jsonBytes)
```

Why not return the error? Instead of ignoring